### DISCUSSION TITLE
无法下载，提示userid=None

### DISCUSSION THREAD

**Discussion by harlonjo** [jxxghp/MoviePilot#1293] [问题咨询]
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 站点搜索进度：3 / 3 |  
-- | -- | -- | --
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 站点搜索完成，有效资源数：2，总耗时 1 秒
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 开始匹配，总 2 个资源 ...
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 标题：鸣龙少年，原标题：鸣龙少年，别名：['The   Hope', '鸣龙少年', 'Thiếu Niên Minh Long', '鳴龍少年', 'The Hope (2023)   ซับไทย']
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 鸣龙少年 通过副标题匹配到资源：铂金学院 - The   Hope 2023 S01E10 2160p HQ WEB-DL 60fps H.265 10bit AAC-YingWEB，副标题：鸣龙少年/中国版龙樱   第一季 第10集 \| 类型:剧情 主演:张若昀/黄尧/王锵 [国语\|内嵌中字]
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 鸣龙少年 通过标题匹配到资源：CARPT -   鸣龙少年
stdout | 2023-12-31 08:29:06 | �[32mINFO:�[0m    search.py - 匹配完成，共匹配到 2 个资源
stdout | 2023-12-31 08:29:06 | �[32mINFO�[0m:     192.168.99.140:0 - "�[1mGET   /api/v1/search/media/tmdb%3A240880?mtype=%E7%94%B5%E8%A7%86%E5%89%A7&area=title   HTTP/1.1�[0m" �[32m200 OK�[0m
stdout | 2023-12-31 08:29:32 | �[31mERROR:�[0m   download.py - 鸣龙少年 (2023) 添加下载任务失败：The   Hope 2023 S01E10 2160p HQ WEB-DL 60fps H.265 10bit AAC-YingWEB -   https://ptchina.org/download.php?id=12319，无法连接qbittorrent下载器
stdout | 2023-12-31 08:29:32 | �[32mINFO:�[0m    chain -   发送消息：channel=None，title=添加下载任务失败：鸣龙少年 (2023) S01 E10, text=站点：铂金学院
stdout | 2023-12-31 08:29:32 | 种子名称：The Hope 2023 S01E10 2160p HQ   WEB-DL 60fps H.265 10bit AAC-YingWEB
stdout | 2023-12-31 08:29:32 | 错误信息：无法连接qbittorrent下载器，userid=None
stdout | 2023-12-31 08:29:32 | �[32mINFO:�[0m    command.py - 处理事件：notice.message -   []
stdout | 2023-12-31 08:29:37 | �[32mINFO�[0m:     192.168.99.140:0 - "�[1mPOST   /api/v1/download/ HTTP/1.1�[0m" �[32m200 OK�[0m
stdout | 2023-12-31 08:29:41 | �[32mINFO�[0m:     192.168.99.140:0 - "�[1mGET   /api/v1/system/logging?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDQ3MDE3ODAsInN1YiI6IjEifQ.7WqkajSf3ZV1XN1EQXdHl3CM6wmm8aRnU1PJWKVzXio   HTTP/1.1�[0m" �[32m200 OK�[0m
stdout | 2023-12-31 08:30:47 | �[32mINFO:�[0m    transfer.py - 开始执行下载器文件转移 ...
stdout | 2023-12-31 08:30:47 | �[32mINFO:�[0m    transfer.py - 没有获取到已完成的下载任务
stdout | 2023-12-31 08:34:12 | �[31mERROR:�[0m   download.py - 鸣龙少年 (2023) 添加下载任务失败：The   Hope 2023 S01E10 2160p HQ WEB-DL 60fps H.265 10bit AAC-YingWEB -   https://ptchina.org/download.php?id=12319，无法连接qbittorrent下载器
stdout | 2023-12-31 08:34:12 | �[32mINFO:�[0m    chain -   发送消息：channel=None，title=添加下载任务失败：鸣龙少年 (2023) S01 E10, text=站点：铂金学院
stdout | 2023-12-31 08:34:12 | 种子名称：The Hope 2023 S01E10 2160p HQ   WEB-DL 60fps H.265 10bit AAC-YingWEB
stdout | 2023-12-31 08:34:12 | 错误信息：无法连接qbittorrent下载器，userid=None |   |   |   |   |  
stdout | 2023-12-31 08:34:12 | �[32mINFO:�[0m    command.py - 处理事件：notice.message -   []
stdout | 2023-12-31 08:34:17 | �[32mINFO�[0m:     192.168.99.140:0 - "�[1mPOST   /api/v1/download/ HTTP/1.1�[0m" �[32m200 OK�[0m
stdout | 2023-12-31 08:34:21 | �[32mINFO�[0m:     192.168.99.140:0 - "�[1mGET   /api/v1/system/logging?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDQ3MDE3ODAsInN1YiI6IjEifQ.7WqkajSf3ZV1XN1EQXdHl3CM6wmm8aRnU1PJWKVzXio   HTTP/1.1�[0m" �[32m200 OK�[0m

**Comment by lu8105** [comment]
一样的问题，楼主解决了吗

**Reply by handsomechao66** [reply]
同问
